user/hasher_test.go

```diff
@@ -22,7 +22,7 @@ func TestGenerateUniqueHashLength(t *testing.T) {
 
 	if theHash, err := generateUniqueHash([]string{"one", "two", "miss a few", "99", "100"}, 20); err == nil {
 		if len(theHash) != 20 {
-			t.Fatalf("the hash should be 20 characters in length but instead is %d", len(theHash))
+			t.Fatalf("the has should be 20 characters in length, but has %v", len(theHash))
```

Comment:
Why removing the "h" from the Hash?

Reply:
This was in the commit from tidepool. Didn't check this.